src/main/java/betterquesting/client/gui2/GuiQuestSearch.java

```diff
@@ -58,16 +59,27 @@ private void createSearchBox(CanvasEmpty cvInner) {
         PanelTextField<String> searchBox = new PanelTextField<>(new GuiTransform(GuiAlign.TOP_EDGE, new GuiPadding(0, 16, 8, -32), 0), "", FieldFilterString.INSTANCE);
         searchBox.setWatermark("Search...");
         searchBox.lockFocus(true);
+        searchBox.enableClearingOnRightClick(true);
         cvInner.addPanel(searchBox);
 
         CanvasQuestSearch canvasQuestSearch = createSearchCanvas();
         cvInner.addPanel(canvasQuestSearch);
 
         searchBox.setCallback(canvasQuestSearch::setSearchFilter);
```

Comment:
I think this line can be deleted, as it is now redundant with the next statement

Reply:
Yes, oversight on my part.